worldedit-sponge/build.gradle

```diff
@@ -0,0 +1,99 @@
+apply plugin: 'eclipse'
```

Comment:
Could we merge some of this with the `worldedit-forge` build.gradle? I'll do it if you're open to the idea.